src/main/java/frc/robot/subsystems/ElevatorIOVortex.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkFlex;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkFlexConfig;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+
+public class ElevatorIOVortex implements ElevatorIO {
+    
+    private final SparkFlex elevatorMotorLeader = new SparkFlex(ElevatorConstants.leaderCAN, MotorType.kBrushless);
+    private final SparkFlex elevatorMotorFollower = new SparkFlex(ElevatorConstants.followerCAN, MotorType.kBrushless);
+
+    public ElevatorIOVortex () {
+
+        SparkFlexConfig config = new SparkFlexConfig();
+
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(ElevatorConstants.currentLimit).voltageCompensation(12.0);
+
+        elevatorMotorLeader.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        elevatorMotorFollower.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+    }
+
+    @Override
+    public void updateInputs (ElevatorIOInputs inputs) {
+
+        inputs.topVoltage = elevatorMotorLeader.getAppliedOutput() * elevatorMotorLeader.getBusVoltage();
+        inputs.topCurrent = elevatorMotorLeader.getOutputCurrent();
+
+        inputs.bottomVoltage = elevatorMotorFollower.getAppliedOutput() * elevatorMotorFollower.getBusVoltage();
+        inputs.bottomCurrent = elevatorMotorFollower.getOutputCurrent();
+    }
+
+    @Override
+    public void move (double volts) {
+
+        elevatorMotorLeader.setVoltage(volts);
+        elevatorMotorFollower.setVoltage(-volts);
```

Comment:
Since the follower motor will automatically follow and the configuration will specify which motor is reversed, you don't need this line.